.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+*PR Title should start with one of the following tags: [Fix]/[Feature]/[Style]/[Update]*
+
+### Description
+*Describe the issue, update, change or fix and notable **why***
+
+### Collateral (docs, reports, design examples, case IDs):
+
```

Comment:
Add:
* Add the IPS case # or HSD # that corresponds to this fix or feature.  Additionally, if you believe this feature needs to be documented in customer facing collateral, please note: "Documentation required" and area affected, for example:
FIM- Documentation Required
AFU- Documentation Required
Scripts- Documentation Required

"I don't believe we want to give links to HAS or internal reports here since we may make commit public at some point"

Reply:
I can add a section on document required however

Reply:
A pox to anyone putting private issue tracker numbers on "open source" public development materials.

To anyone consuming the repository who's not an Intel employee, it's at best of zero value. You're hiding the context that brought about the change.

Reply:
Have we considered what kind of public bug tracking we will have for open-source projects?